tests/storage/memorydump.go

```diff
@@ -611,4 +613,104 @@ var _ = SIGDescribe("Memory dump", func() {
+			By("Invoking virtctl memory dump download")
+			commandAndArgs := []string{commandMemoryDump, "download", name, virtCtlNamespace, namespace}
+			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlOutputFile, outputFile))
+			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
+			Eventually(func() error {
+				return memorydumpCommand()
+			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
+		}
+
+		memoryDumpVirtctlGetWithDownload := func(name, namespace, claimName, outputFile string) {
+			By("Invoking virtctl memory dump")
+			commandAndArgs := []string{commandMemoryDump, "get", name, virtCtlNamespace, namespace}
+			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlClaimName, claimName))
+			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlOutputFile, outputFile))
+			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
+			Eventually(func() error {
+				return memorydumpCommand()
+			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
+		}
+
+		memoryDumpVirtctlCreateWithDownload := func(name, namespace, claimName, outputFile string) {
+			By("Invoking virtctl memory dump with create flag")
+			commandAndArgs := []string{commandMemoryDump, "get", name, virtCtlNamespace, namespace}
+			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlClaimName, claimName))
+			commandAndArgs = append(commandAndArgs, virtCtlCreate)
+			commandAndArgs = append(commandAndArgs, fmt.Sprintf(virtCtlOutputFile, outputFile))
+			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
+			Eventually(func() error {
+				err := memorydumpCommand()
+				if err != nil {
+					_, getErr := virtClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.Background(), claimName, metav1.GetOptions{})
+					if getErr == nil {
+						// already created the pvc can't call the memory dump command with
+						// create-claim flag again
+						By("Error memory dump command after claim created")
+						memoryDumpVirtctlGetWithDownload(name, namespace, claimName, outputFile)
+						return nil
+					}
+				}
+				return err
+			}, 20*time.Second, 2*time.Second).ShouldNot(HaveOccurred())
+		}
+
+		BeforeEach(func() {
+			if !checks.IsOpenShift() {
+				Skip("Need ingress to run this test which we only have on openshift")
+			}
+			sc, exists := libstorage.GetRWOFileSystemStorageClass()
+			if !exists {
+				Skip("Skip no filesystem storage class available")
+			}
+			libstorage.CheckNoProvisionerStorageClassPVs(sc, numPVs)
+
+			vm = createAndStartVM()
+			outputFile = fmt.Sprintf(defaultOutput, rand.String(12))
+		})
+
+		AfterEach(func() {
+			if vm != nil {
+				deleteVirtualMachine(vm)
+			}
+			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(util.NamespaceTestDefault).Get(context.Background(), memoryDumpPVCName, metav1.GetOptions{})
+			if err == nil && pvc != nil {
+				deletePVC(pvc)
+			}
+			err = os.Remove(outputFile)
+			if err != nil {
```

Comment:
Could be cleaner to do something like this:
```go
if _, err := os.Remove(outputFile); !errors.Is(err, os.ErrNotExist) {
  Fail(err)
}
```

Reply:
done, with nil handling ;)